From bunk@stusta.de Sat Mar 25 09:03:43 2006
Date: Sat, 25 Mar 2006 18:03:38 +0100
From: Adrian Bunk <bunk@stusta.de>
To: gregkh@suse.de
Subject: USB: input/: proper prototypes
Message-ID: <20060325170338.GF4053@stusta.de>
Content-Disposition: inline

This patch adds proper prototypes in a header file for some global
functions.

Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/input/hid-ff.c |    6 ------
 drivers/usb/input/hid.h    |    5 +++++
 2 files changed, 5 insertions(+), 6 deletions(-)

--- gregkh-2.6.orig/drivers/usb/input/hid-ff.c
+++ gregkh-2.6/drivers/usb/input/hid-ff.c
@@ -34,12 +34,6 @@
 
 #include "hid.h"
 
-/* Drivers' initializing functions */
-extern int hid_lgff_init(struct hid_device* hid);
-extern int hid_lg3d_init(struct hid_device* hid);
-extern int hid_pid_init(struct hid_device* hid);
-extern int hid_tmff_init(struct hid_device* hid);
-
 /*
  * This table contains pointers to initializers. To add support for new
  * devices, you need to add the USB vendor and product ids here.
--- gregkh-2.6.orig/drivers/usb/input/hid.h
+++ gregkh-2.6/drivers/usb/input/hid.h
@@ -533,3 +533,8 @@ static inline int hid_ff_event(struct hi
 		return hid->ff_event(hid, input, type, code, value);
 	return -ENOSYS;
 }
+
+int hid_lgff_init(struct hid_device* hid);
+int hid_tmff_init(struct hid_device* hid);
+int hid_pid_init(struct hid_device* hid);
+
